### PR TITLE
Added more Instaclustr links

### PR DESCRIPTION
**Pull request by smiklosovic**
Hi @polandll ,

Instaclustr here.

would you mind to add these links for us into third party section?

Thanks!

### CODE REVIEW

**Comment by michaelsembwever** [comment]
LGTM 👍 
 thanks @smiklosovic for the initiative!

@polandll does it look ok to you?

File: src/third-party.md [hunk 1]
```diff
@@ -50,7 +60,7 @@ Snap it into your existing workflows with the click of a button, automate away t
...
 * [DataStax Management API for Apache Cassandra](https://github.com/datastax/management-api-for-apache-cassandra): RESTful / Secure Management Sidecar for Apache Cassandra
 * [DataStax Spring Boot](https://github.com/datastax/spring-boot): Spring Boot extension
-* [Instaclustr cassandra-sidecar](https://github.com/instaclustr/cassandra-sidecar): This repository is home of a sidecar for Apache Cassandra database. Sidecar is meant to be run alongside of Cassandra instance and sidecar talks to Cassandra via JMX.
+* [Instaclustr Icarus](https://github.com/instaclustr/icarus): This repository is home of a sidecar for Apache Cassandra database. Icarus is meant to be run alongside of Cassandra instance and it talks to Cassandra via JMX. It embeds [Instaclustr Esop](https://github.com/instaclustr/esop) so you can backup and restore your cluster on-the-fly to and from various cloud storage providers. Icarus is used as a container in [Instaclustr cassandra-operator](https://github.com/instaclustr/cassandra-operator) as well as in [Orange CassKop](https://orange-opensource.github.io/casskop/) where it is used for backup and restoration purposes primarily.
```
Comment (polandll) [inline src/third-party.md]: Could you please tighten up the language? 
Icarus is meant to be run alongside of Cassandra, talking to Cassandra via JMX. [Instaclustr Esop](https://github.com/instaclustr/esop) is embedded in Icarus for on-the-fly cluster backup and restore using various cloud storage providers as a source or destination. Icarus is used primarily as a backup and restore tool, and is containerized in [Instaclustr cassandra-operator](https://github.com/instaclustr/cassandra-operator) and [Orange CassKop](https://orange-opensource.github.io/casskop/).

File: src/third-party.md [hunk 2]
```diff
@@ -36,6 +36,16 @@ Snap it into your existing workflows with the click of a button, automate away t
 * [The Last Pickle Cassandra stress tool, tlp-stress](https://github.com/thelastpickle/tlp-stress): A workload-centric stress tool for Apache Cassandra. Designed for simplicity, no math degree required. (DataStax)
 * [NoSQLBench](https://github.com/nosqlbench/nosqlbench): Pluggable benchmarking suite for Cassandra and other distributed systems
 * [Stargate](https://stargate.io): Open source data gateway providing Document, REST, and GraphQL APIs to Apache Cassandra.
+* [Instaclustr Esop](https://github.com/instaclustr/esop): Swiss knife for backup and restore of your node to GCP, Azure, S3, Ceph etc. Supports backup and restoration of commit logs too. Esop is embedded in [Instaclustr Icarus](https://github.com/instaclustr/icarus) sidecar so you may backup and restore your cluster remotely and on-the-fly with any disruption.
```
Comment (polandll) [inline src/third-party.md]: Esop is embedded in [Instaclustr Icarus](https://github.com/instaclustr/icarus) sidecar for remote cluster on-the-fly backup and restore with any disruption.

And is that supposed to be "without any disruption"?

File: src/third-party.md [hunk 3]
```diff
@@ -36,6 +36,16 @@ Snap it into your existing workflows with the click of a button, automate away t
...
+* [Instaclustr Minotaur](https://github.com/instaclustr/instaclustr-minotaur): Command line tool for consistent rebuilding of a Cassandra cluster.
+* [Instaclustr LDAP Authenticator](https://github.com/instaclustr/cassandra-ldap): LDAP Authenticator for Apache Cassandra
+* [Instaclustr TTL Remover](https://github.com/instaclustr/cassandra-ttl-remover): Command line tool for rewriting SSTables to remove TTLs
+* [Instaclustr SSTable Generator](https://github.com/instaclustr/cassandra-sstable-generator): CLI tool for programmatic generation of Cassandra SSTables
+* [Instaclustr Exporter](https://github.com/instaclustr/cassandra-exporter): Java agent that exports Cassandra metrics to Prometheus
+* [Instaclustr Go Client for Instaclustr Icarus](https://github.com/instaclustr/instaclustr-icarus-go-client): Go client for Instaclustr Icarus sidecar
+* [Instaclustr SStable Tools](https://github.com/instaclustr/cassandra-sstable-tools): A command line tool that helps admins get summaries, metadata, partition info, cell info for SSTables
```
Comment (polandll) [inline src/third-party.md]: [Instaclustr SStable Tools](https://github.com/instaclustr/cassandra-sstable-tools): A command line tool that helps admins get summaries, metadata, partition info, and cell info for SSTables.

File: README.md [hunk 4]
```diff
@@ -42,6 +42,8 @@ $ docker-compose run cassandra-website
 
 :warning: *Tip:* In order to prevent root-owned modified files in your repository, the container user, `build`, is set up with a default UID=1000:GID=1000, which is usually the first user configured on a linux machine. If your local user is different you should set up the container user with your local host user's UID:GID, replace the above with:
 
+:warning: *Tip:* Building of cassandra-website may fail on not having `CLOUDSDK_PYTHON` variable set at your machine. Set that environment variable for example like `export CLOUDSDK_PYTHON=/usr/bin/python2`.
```
Comment (polandll) [inline README.md]: Building cassandra-website may fail if the `CLOUDSDK_PYTHON` environment variable is not set on your machine. For example, set the environment variable to `export CLOUDSDK_PYTHON=/usr/bin/python2`.

**Review by polandll — COMMENTED** [review]
I've left some comments for review, to make the writing on this page more consistent.

**Comment by smiklosovic** [comment]
@polandll thanks for the feedback, I am not a native speaker so it might be little bit harsh :) I hope it is ok now. Feel free to squash.

**Comment by michaelsembwever** [comment]
Merged with https://github.com/apache/cassandra-website/commit/e18f93d25bc0c50a1a77145123fefccc94351eff

**Comment by michaelsembwever** [comment]
On https://cassandra.staged.apache.org/third-party/

**Comment by michaelsembwever** [comment]
Live at https://cassandra.apache.org/third-party/

**Pull request closed without merging**